feature.env

```diff
@@ -13,4 +13,6 @@ TOKENVERIFICATION_API_URL=http://localhost:18100
 ALLOCATION_MANAGER_ENDPOINT_URL=http://localhost:18084/
 TOKENVERIFICATION_API_ENABLED=true
 OFFENDER_SEARCH_API_URL=http://localhost:18101
+OFFENDER_SEARCH_API_ENABLED=true
```

Comment:
you've enabled offender search by accident

Reply:
Good spot there! Cheers Pete :)

Reply:
Seems like it needs the true value to run the tests - but gotten rid of the duplicate false value at the bottom 👍

Reply:
no it needs the `false` value :confused:

Reply:
Ahh may have been failing because there was a duplicate then, works with the false value :)

Reply:
so please remove the duplicate